examples/autocomplete/option-highlight.js

```diff
@@ -9,7 +9,7 @@ const { prompt } = require('enquirer');
     type: 'autocomplete',
     name: 'flavor',
     message: 'Pick your favorite flavor',
-    highlight: colors.green,
+    highlight: text => colors.green(text),
```

Comment:
```suggestion
    highlight: str => colors.green(str),
```
I think using `str` instead of `text` will keep this example consistent with other examples.